resources/js/Pages/Admin/BookingRequests/UpdateBookingRequestForm.vue

```diff
@@ -95,18 +104,21 @@ export default {
 
     data() {
         return {
-            availableExcludingCurrent: [],
+            availableExcludingCurrent: [], 
             form: this.$inertia.form(
-                {
+                { 
                     user_id: null,
                     room_id: null,
                     start_time: null,
                     end_time: null,
+                    reference: [],
+                    _method: 'PUT'
```

Comment:
Can't seem to find where _method is being used

Reply:
This is Laravel's method spoofing mechanism. In order to send an array of files to the controller via a PUT request we must pass this hidden "_method" variable through a POST.